mkdocs.yml

```diff
@@ -299,8 +299,12 @@ extra:
     version:
         provider: mike
     analytics:
-        provider: google
-        property: G-3M31G9B0QJ
+        provider: custom
+        property: !ENV [AMPLITUDE_API_KEY, "8ab9ae57b486baa5d4ab626559fd8106"]
```

Comment:
Is this key something we can inject from the netlify runner instead? Weird to see this hard coded

Reply:
We are, it's purely here for testing locally but it makes sense to remove it. Will drop this now